client/my-sites/subscribers/components/subscriber-totals/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { translate } from 'i18n-calypso';
+import { SubscribersFilterBy } from '../../constants';
+import './style.scss';
+
+type SubscriberTotalsProps = {
+	totalSubscribers: number;
+	filteredCount: number;
+	filterOption: SubscribersFilterBy;
+	searchTerm: string;
+	isLoading: boolean;
+};
+
+const getFilterLabel = ( filterOption: SubscribersFilterBy, count: number ): string => {
+	switch ( filterOption ) {
+		case SubscribersFilterBy.Paid:
+			return count === 1 ? translate( 'paid subscriber' ) : translate( 'paid subscribers' );
+		case SubscribersFilterBy.Free:
+			return count === 1 ? translate( 'free subscriber' ) : translate( 'free subscribers' );
+		case SubscribersFilterBy.EmailSubscriber:
+			return count === 1 ? translate( 'email subscriber' ) : translate( 'email subscribers' );
+		case SubscribersFilterBy.ReaderSubscriber:
+			return count === 1 ? translate( 'reader subscriber' ) : translate( 'reader subscribers' );
+		default:
+			return count === 1 ? translate( 'subscriber' ) : translate( 'subscribers' );
+	}
+};
+
+const SubscriberTotals: React.FC< SubscriberTotalsProps > = ( {
+	totalSubscribers,
+	filteredCount,
+	filterOption,
+	searchTerm,
+	isLoading,
+} ) => {
+	if ( isLoading ) {
+		return <div className="subscriber-totals is-loading" />;
+	}
+
+	const isFiltered = filterOption !== SubscribersFilterBy.All || !! searchTerm;
+	const filterLabel = getFilterLabel( filterOption, filteredCount );
+
+	return (
+		<div className="subscriber-totals">
+			{ isFiltered ? (
+				<>
+					<span className="subscriber-totals__filtered-count">
+						{ searchTerm
+							? translate( '%(count)d matching result', '%(count)d matching results', {
+									count: filteredCount,
+									args: { count: filteredCount },
```

Comment:
These counts must go through `numberFormat` method to be properly localised. Otherwise we end up with commas used in place of dots and vice versa. Can you please create an issue for all these cases so we can fix? Please tag team @Automattic/i18n on the issue so we can move it under the i18n-issues repo.

Reply:
And we will need to update the variable to be a string, so we will be requesting new translations. Also this will need new variable names to avoid the translation becoming fuzzy and leading to errors with strings injected for decimals. 🤷 

There is a tool that we use for checking the similarity of strings, which will be handy. Please ping me in the issue to share.

Reply:
TIL! I'm not clear on the next step here. Should I open an issue or a PR? Is there any documentation I can follow?

Reply:
I think next steps should be to create an issue, with some screenshots of where the error occurs. Try with a value greater than 1000 and perhaps different locales, it should render with  the correct separators (commas in en, dots in EL/DE, etc.)

Reply:
I found one example and opened a PR: https://github.com/Automattic/wp-calypso/pull/100610 I tagged the i18n team there.